includes/class-sensei.php

```diff
@@ -1418,29 +1413,33 @@ public function get_settings_url( $plugin_id = null ) {
 		return admin_url( 'admin.php?page=sensei-settings&tab=general' );
 	}
 
-		/**
-		 * Gets the plugin documentation url, used for the 'Docs' plugin action
-		 *
-		 * @return string documentation URL
-		 */
+	/**
+	 * Gets the plugin documentation url, used for the 'Docs' plugin action
+	 *
+	 * @return string documentation URL
+	 */
 	public function get_documentation_url() {
-		return sprintf( 'https://docs.woothemes.com/documentation/plugins/sensei/?utm_source=SenseiPlugin&utm_medium=PluginPage&utm_content=Docs&utm_campaign=SenseiPlugin' );
+		return sprintf( 'https://senseilms.com/documentation/' );
 	}
 
-		/**
-		 * Gets the support URL, used for the 'Support' plugin action link
-		 *
-		 * @return string support url
-		 */
+	/**
+	 * Gets the support URL, used for the 'Support' plugin action link
+	 *
+	 * @deprecated 3.1.1
+	 *
+	 * @return string support url
+	 */
 	public function get_support_url() {
+		_deprecated_function( __METHOD__, '3.1.1' );
```

Comment:
Have we added this to the appropriate issue in the version where it should be removed?

Reply:
Added now! ;)